src/Uno.UI/ts/Windows/System/Profile/AnalyticsInfo.ts

```diff
@@ -0,0 +1,59 @@
+﻿namespace Windows.System.Profile {
+
+	export class AnalyticsInfo {
+		public static getDeviceType(): string {
```

Comment:
Is there a reference source for those checks ?

Reply:
Added link to source